dataformat-json-jackson/src/main/java/org/camunda/spin/impl/json/jackson/JacksonJsonNode.java

```diff
@@ -154,6 +154,24 @@ public Integer lastIndexOf(Object searchObject) {
     }
   }
 
+  public boolean contains(Object searchObject) {
```

Comment:
What do you think about reusing the `indexOf` method here instead of implementing similar logic?
If an index is returned the array contains the search object. If an exception is thrown it does not.
Happy to discuss.